src/types/onyx/ReportAction.ts

```diff
@@ -151,10 +171,13 @@ type ReportActionBase = OnyxCommon.OnyxValueWithOfflineFeedback<{
     /** Whether we have received a response back from the server */
     isLoading?: boolean;
 
+    /** Avatar data to display on the report action */
     avatar?: AvatarSource;
 
+    /** TODO: not enough context, seems to be used in tests only */
     automatic?: boolean;
 
+    /** TODO: Not enough context, seems to be used in tests only */
     shouldShow?: boolean;
```

Comment:
I'm pretty sure we could remove both, but it would take a lot of refactor in `ReportUtils.ts` and tests. How about marking these as `@depracated` with explanation?